### PR TITLE
v3 farming ui

### PR DESCRIPTION
**Pull request by 0xModene**
# **Pull Request Template**

###### *Select a category that relates to the content of your pull request (choose all that apply).*

- [ ] Bug Fix
- [ ] Content
- [x] New Feature
- [ ] Documentation
- [ ] Code Refactoring

&nbsp;

###### *Provide a brief summary of the changes you made and their purpose as they relate to Index UI. <br/>If applicable, please link the corresponding issue number.*

&nbsp;

### **Fixes: Issue #**

&nbsp;

###### *Include proof that your changes function properly and resolve the linked issue.*

&nbsp;

###### *By submitting this pull request, you are confirming the following to be true:*

- I have reviewed the [Contribution Guidelines](https://github.com/SetProtocol/index-ui/blob/master/CONTRIBUTING.md).
- I have performed a self-review of my own code.
- I have updated my repository to match the master at `SetProtocol/index-ui`.
- I have included test data or screenshots that prove my fix is effective or that my feature works.

### CODE REVIEW

File: src/views/Farm/components/Stake/DpiFarm.tsx [hunk 1]
```diff
@@ -1,129 +1,111 @@
-import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import React, { useCallback, useMemo, useState } from 'react'
```
Comment (controtie) [inline src/views/Farm/components/Stake/DpiFarm.tsx:1]: @0xModene I'm assuming this file is just copy/pasted from the old implementation to restore it?

**Review by 0xModene — COMMENTED** [review]
lgtm

File: src/views/Farm/components/Stake/DpiFarm.tsx (hunk 1 above)
Comment (0xModene) [reply src/views/Farm/components/Stake/DpiFarm.tsx:1]: Yes, since I copied ncitron's changes wholly, this was needed.

**Review by 0xModene — COMMENTED** [review]
lgtm, added 3 hook dependencies but that's it

**Pull request merged**